Swiftfin/Views/LibraryListView.swift

```diff
@@ -73,36 +73,32 @@ struct LibraryListView: View {
 					}
 
 					ForEach(viewModel.libraries, id: \.id) { library in
-						if library.collectionType ?? "" == "movies" || library.collectionType ?? "" == "tvshows" {
```

Comment:
Don't re-implement this exact filter, but we need a filter of some kind so that we only show expected libraries (ie: not music libraries)

Reply:
I also believe that you can delete the explicit check that we have for the "Collections" library, as this will catch it instead.